offchain/advance-runner/src/config.rs

```diff
@@ -62,7 +62,9 @@ pub enum ConfigError {
     SnapshotConfigError { source: SnapshotConfigError },
 }
 
-#[derive(Parser, Debug)]
+#[derive(Clone, Parser)]
```

Comment:
Do we need the clone?

Reply:
Some use case needed, not sure why. I was just making it standard. Will remove and re-check who really needs